packages/reactivity/src/ref.ts

```diff
@@ -3,11 +3,11 @@ import { OperationTypes } from './operations'
 import { isObject } from '@vue/shared'
 import { reactive } from './reactive'
 
-export const refSymbol = Symbol()
+export const refSymbol = Symbol('isRef')
 export type RefSymbol = typeof refSymbol
 
 export interface Ref<T> {
-  _isRef: RefSymbol
+  [refSymbol]: boolean
```

Comment:
This can only be `true` imho:
```suggestion
  [refSymbol]: true
```